### PR TITLE
Add unittests for AWS snapshoting capabilities.

### PR DESCRIPTION
**Pull request by giovannt0**
Signed-off-by: Theo Giovanna <gtheo@google.com>

### CODE REVIEW

File: tests/unittest/aws_test.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+}
+
+
+class AWSAccountTest(unittest.TestCase):
+  """Test AWSAccount class."""
+
+  @mock.patch('libcloudforensics.aws.AWSAccount.ClientApi')
+  def testListInstances(self, mock_ec2_api):
+    """Test that instances of an account are correctly listed."""
+    mock_ec2_api.return_value.describe_instances.return_value = MOCK_DESCRIBE_INSTANCES  # pylint: disable=line-too-long
```
Comment (tomchop) [inline tests/unittest/aws_test.py:155]: You can use `pylint` statements at the class or function level, which might make it easier to read

Comment (tomchop) [inline tests/unittest/aws_test.py:155]: Have you tried assigning `mock_ec2_api.return_value.describe_instances` to its own variable? Might make the code more readable if you reuse it below, but I don't know how that will affect the mock behavior

File: tests/unittest/aws_test.py [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+    self.assertRaises(ValueError, FAKE_AWS_ACCOUNT.GetVolumesByNameOrId)
+    self.assertRaises(
+        ValueError,
+        FAKE_AWS_ACCOUNT.GetVolumesByNameOrId,
+        volume_id=FAKE_VOLUME.volume_id,
+        volume_name=FAKE_BOOT_VOLUME.name)
+
+  @mock.patch('libcloudforensics.aws.AWSAccount.ClientApi')
+  def testCreateDiskFromSnapshot(self, mock_ec2_api):
+    """Test the creation of a volume from a snapshot."""
```
Comment (tomchop) [inline tests/unittest/aws_test.py:288]: If you're testing the creation of a Volume, should the test function be renamed for consistency?

File: tests/unittest/aws_test.py (hunk 1 above)
Comment (giovannt0) [reply tests/unittest/aws_test.py:155]: Gotta love Python, this works well. I've also migrated the other mock statements to their own variable regardless of their re-use or not (only for 80+ chars lines), as I would prefer not to disable `pylint` at class/method level.

**Pull request merged**